net-im/slack-bin/slack-bin-2.3.3.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit unpacker gnome2-utils
+
+QA_PREBUILT="
+	opt/slack/slack
+	opt/slack/resources/app.asar.unpacked/node_modules/*
+	opt/slack/libnode.so
+	opt/slack/libgcrypt.so.11
+	opt/slack/libffmpeg.so
+	opt/slack/libCallsCore.so
+"
+
+DESCRIPTION="Team collaboration tool"
+HOMEPAGE="http://www.slack.com/"
+
+MY_PN="${PN/-bin/}"
+BASE_URI="https://downloads.slack-edge.com/linux_releases/${MY_PN}-desktop-${PV}-_arch_.deb"
+
+SRC_URI="
+	amd64? ( ${BASE_URI/_arch_/amd64} )
```

Comment:
Do you still need to keep the BASE_URI play once upstream no longer supports more than one arch?